test/integration/targets/lightsail/tasks/main.yml

```diff
@@ -0,0 +1,136 @@
+---
+
+- name: Integration test for lightsail module
+  block:
+
+    # ==== Setup ==================================================
+
+    - name: Set connection information for all tasks
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{ aws_access_key }}"
+          aws_secret_key: "{{ aws_secret_key }}"
+          security_token: "{{ security_token }}"
+          region: "{{ aws_region }}"
+      no_log: yes
```

Comment:
As an FYI, we're trying to move more towards using module_defaults for credentials in new tests.  This works just fine for this PR though.
https://docs.ansible.com/ansible/devel/dev_guide/platforms/aws_guidelines.html#aws-credentials-for-integration-tests

Reply:
I can make that change to use module_defaults.